flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/TemporalJoinITCase.scala

```diff
@@ -455,15 +455,12 @@ class TemporalJoinITCase(state: StateBackendMode)
       " ON o.currency = r.currency"
 
     tEnv.executeSql(sql).await()
-    val rawResult = getRawResults("rowtime_default_sink")
     val expected = List(
-      "+I(1,Euro,12,2020-08-15T00:01,114,2020-08-15T00:00:01)",
-      "+I(2,US Dollar,1,2020-08-15T00:02,102,2020-08-15T00:00:02)",
-      "+I(3,RMB,40,2020-08-15T00:03,702,2020-08-15T00:00:04)",
-      "+I(4,Euro,14,2020-08-16T00:04,118,2020-08-16T00:01)",
-      "-U(2,US Dollar,1,2020-08-16T00:03,106,2020-08-16T00:02)",
-      "+U(2,US Dollar,18,2020-08-16T00:03,106,2020-08-16T00:02)")
-    assertEquals(expected.sorted, rawResult.sorted)
+      "1,Euro,12,2020-08-15T00:01,114,2020-08-15T00:00:01",
+      "3,RMB,40,2020-08-15T00:03,702,2020-08-15T00:00:04",
+      "4,Euro,14,2020-08-16T00:04,118,2020-08-16T00:01",
+      "2,US Dollar,18,2020-08-16T00:03,106,2020-08-16T00:02")
```

Comment:
```suggestion
      "1,Euro,12,2020-08-15T00:01,114,2020-08-15T00:00:01",
      "2,US Dollar,18,2020-08-16T00:03,106,2020-08-16T00:02",
      "3,RMB,40,2020-08-15T00:03,702,2020-08-15T00:00:04",
      "4,Euro,14,2020-08-16T00:04,118,2020-08-16T00:01")
```
minor： we can give a more readable order that is same as the left table input.